application-accelerator/provenance.hbs.md

```diff
@@ -21,3 +21,54 @@ project:
 
 For more information about the structure of the file and how to enable application bootstrapping
 provenance, see [Provenance transform](creating-accelerators/transforms/provenance.hbs.md).
+
+## <a id="amr"></a> Integration with AMR
+
+Application Accelerator integrates with [Artifact Metadata Repository](../scst-store/overview.hbs.md) (AMR).
+
+When you generate an application with an accelerator, an event that contains the same information captured by the `Provenance` transform is sent to the AMR store.
+
+The relevant data saved to AMR are:
+
+### <a id='appacceleratorruns'></a> AppAcceleratorRuns (alpha)
+
+The `AppAcceleratorRuns` data model represents new projects running from Git
+repositories. An `accelerator.yaml` file in the repository declares input
+options for the accelerator. This file contains instructions for processing the
+files when you generate a new project. Observer sends this information to the
+CloudEvent Handler to store `AppAcceleratorRuns`.
+
+Each `AppAcceleratorRun` data entry has a unique `guid`. The `guid` includes information
+about the Git repository including, `AppAcceleratorRepoURL`, `AppAcceleratorRevision`,
+and `AppAcceleratorSubpath`. You can point multiple `AppAcceleratorFragments` entries
```

Comment:
Who is "you" in this sentence? Data is captured at the time of project generation, and then this doc should be about querying it. I don't understand what "pointing" means here.

Reply:
Woud you suggest removing those sentences? In other words, do they contain information that might be useful to a user?

So remove this: 
"You can point multiple `AppAcceleratorFragments` entries to the same `AppAcceleratorRun` entry. You can also associate an `AppAcceleratorRun`
with one `AppAcceleratorSource`, also known as `Commit`."


If we want to keep this maybe we can reword to:
"Multiple `AppAcceleratorFragments` entries can point to the same `AppAcceleratorRun` entry. `AppAcceleratorRun` are associated
with one `AppAcceleratorSource`, also known as `Commit`."

Reply:
yes; let's reword as you suggest